docs/ONNX_Runtime_Graph_Optimizations.md

```diff
@@ -41,6 +42,15 @@ These optimizations include complex node fusions. They are run after graph parti
 * Matmul Add Fusion
 * Conv Activation Fusion
 * GELU Fusion
+* Layer Normalization Fusion
+
+And BERT related optimizations:
+* Embedding Layer Fusion
+* Attention Fusion
+* Layer Normalization and Skip Connection (with optional Add bias) Fusion
+* GELU Approximiation
+
+Approximation is used in GELU approximation and Attention fusion. There might be slight difference in result after these optimizations.
```

Comment:
Could you elaborate this a little more. For instance add more information on why there could be result diffs and the trade-offs of using this optimization .i.e perf vs accuracy.

Reply:
I will add some comments of accuracy impact when result is available later.

Reply:
I added comment about accuracy impact of approximation. Measurement on SQuAD v1.1 data set shows that the impact can be neglected (F1 score changes from 87.05 to 87.03).